pygmt/src/text.py

```diff
@@ -39,9 +39,6 @@
 @kwargs_to_strings(
     R="sequence",
     textfiles="sequence_space",
-    angle="sequence_comma",
-    font="sequence_comma",
-    justify="sequence_comma",
```

Comment:
These lines were initially added in https://github.com/GenericMappingTools/pygmt/pull/321, but they should be like this.